dev/PipsPager/PipsPager.idl

```diff
@@ -1,33 +1,31 @@
 ﻿namespace MU_XC_NAMESPACE
 {
 
-[WUXC_VERSION_PREVIEW]
+[WUXC_VERSION_MUXONLY]
```

Comment:
Watch out for merge issues with Jevan's PR where he is changing these attribute names.